quickjs.c

```diff
@@ -74,31 +74,6 @@
 #define __extension__
 #endif
 
-// Debug trace system: the debug output will be produced to the dump stream (currently
-// stdout) if qjs is invoked with -D<bitmask> with the corresponding bit set.
-
-#ifndef NDEBUG
-#define DUMP_BYTECODE_FINAL   0x01  /* dump pass 3 final byte code */
-#define DUMP_BYTECODE_PASS2   0x02  /* dump pass 2 code */
```

Comment:
Using a specific define `ENABLE_DUMPS` was fine and preferreable over using `#ifndef NDEBUG` everywhere.
I suppose saghul did not want it defined in the `CMakefile.txt`. You should define here this way:
```
#ifndef NDEBUG
#define ENABLE_DUMPS
#endif
```
This allows temporary definition to investigate problems in the release build.

Sorry for the confusion and the extra work.